source/MaterialXTest/ShaderValid.cpp

```diff
@@ -512,9 +512,14 @@ static void runOSLValidation(const std::string& shaderName, mx::TypedElementPtr
             std::string elementType(element->getType());
             mx::string outputName = element->getName();
 
-            const std::string SURFACE_SHADER("surfaceshader");
+            std::set<std::string> colorClosures =
```

Comment:
Fix for recent addition of closure tests. Any of these are aliases (pre-processor defines) to a "closure color" in OSL.